project1.css

```diff
@@ -29,17 +31,45 @@ section {
   top: 92px;
 }
 
+@media screen and (min-width: 480px) {
```

Comment:
- [ ] Your desktop version seems to fall out of the browser viewport width. I think it's because of the usage of absolute positioning for your project layout. Kindly check out this[ article](https://www.w3schools.com/css/css3_flexbox.asp) on how to use flexbox 👍 .

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/178104422-22c069fb-60d2-49f8-bec5-611077927722.png)  | ![image](https://user-images.githubusercontent.com/63473584/178104379-0b1d4a3e-7b42-4020-9ed3-52d008961e42.png)  |